### PR TITLE
add albums#show and js to change track

### PR DESCRIPTION
**Pull request by sonnv-1936**
==> Title: add albums#show and js to change track

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/13093

### CODE REVIEW

**Comment by sonnv-1936** [comment]
ready

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,4 +12,27 @@
 
 $(document).on(`turbolinks:load`, function () {
```
Comment (huuphuc93) [inline app/assets/javascripts/application.js]: Chuyển dùng ' thay cho ` nhé

File: app/helpers/albums_helper.rb [hunk 2]
```diff
@@ -2,4 +2,17 @@ module AlbumsHelper
   def albums_reference_collection
     Album.pluck :name, :id
   end
+
+  def join_categories categories, css_class = nil
+    html = ""
```
Comment (huuphuc93) [inline app/helpers/albums_helper.rb]:
```suggestion
    categories.map! {|category| link_to(category.name, category, class: css_class)}
    categories.join(", ")
```

**Comment by sonnv-1936** [comment]
ready

@huuphuc93 done ạ :dancer:

**Pull request merged**